src/signer.ts

```diff
@@ -275,6 +294,20 @@ export class Signer {
     return {busy: false};
   }
 
+  private handleOwnerInitialized({data: msgData, origin}: SignerMessageEvent): {
+    initialized: boolean;
+  } {
+    if (isNullish(this.#owner)) {
```

Comment:
Should we use Zod to assert not nullish and no anonymous as well? Just in case sh** happens.